cola/libdialect/io.cpp

```diff
@@ -114,7 +114,9 @@ Graph_SP dialect::buildGraphFromTglf(istream &in) {
             switch(gtc) {
                 case 'B': gt=GapType::BDRY; break;
                 case 'C': gt=GapType::CENTRE; break;
-                COLA_ASSERT(false);
+                default:
+                    COLA_ASSERT(false);
+                    gt=GapType::BDRY;
```

Comment:
Instead of adding `default` clauses, could we just default-initialize the enums where they are declared, like this?

```
            GapType gt{};
            SepDir sd{};
            SepType st{};
```

See https://stackoverflow.com/q/53897991